static/js/publisher/release/reducers/pendingReleases.js

```diff
@@ -68,6 +68,14 @@ function releaseRevision(
 
   if (previousRevisions) {
     state[revision.revision][channel].previousRevisions = previousRevisions;
+
+    if (!progressive) {
```

Comment:
I think that it would be better to keep logic like this in action.

This makes the reducer depend on current time, which makes it less 'pure' and harder to test.
Sure, moving it to action would do the same to the action, but these are generally more complicated anyway.

I guess ideally we would want it out of redux completely and passed by component, but I suppose this would complicate other things.